.ruby-version

```diff
@@ -0,0 +1,2 @@
+ruby-2.2.4
```

Comment:
Hi @ericsaboia, thanks again for your PR!
Could you please update the Ruby version to `2.3.0`? Thanks.